airbyte-integrations/connectors/source-linkedin-ads/source_linkedin_ads/source.py

```diff
@@ -336,6 +336,59 @@ class AdCreativeAnalytics(LinkedInAdsAnalyticsStream):
     pivot_by = "CREATIVE"
 
 
+class AdCreativeName(LinkedinAdsStream):
```

Comment:
Can we re-use the `Creatives` implementation class to inherit from it and override only the necessary parts, like: 
`search_param_value` and `search_param`, the `parent_values_map` would have the value of `{"creative_id": "id"}` that way we will keep the right structure and will not miss any of the campaigns the user has.

Reply:
Yes I think. This would be the best suitable way. I will look into it.

Reply:
I tried it doing with inheriting only the parts `search_param_value`, `search_param`, `parent_values_map` but doesn't work

Reply:
Done.